### DISCUSSION TITLE
Cannot download it properly

### DISCUSSION THREAD

**Discussion by Cedar3702** [seerge/g-helper#778] [Q&A] [answered]
I have extracted the file and when I run the .exefile I get this 
![image](https://github.com/seerge/g-helper/assets/122573966/72de8063-0d31-42b5-af1f-ad78693978ce)
It seems that I have net 7
![image](https://github.com/seerge/g-helper/assets/122573966/81700b3e-bcf5-4b9d-bf6a-6718b2364c4c)

**Comment by seerge** [comment, marked as answer]
@Cedar3702 hello, right click on exe, select properties and check "Unblock" there. It's some strange windows security settings on your laptop that prevent running exe at all.

**Reply by Cedar3702** [reply]
Thank you that worked!
